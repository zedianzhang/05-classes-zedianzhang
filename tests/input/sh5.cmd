:load tests/input/tAdd.hs
((sub ((add 10) 20)) 5)
:quit