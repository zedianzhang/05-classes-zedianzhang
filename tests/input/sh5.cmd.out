------------------------------------------------------------
-------- The NANO Interpreter v.0.0.0.0 --------------------
------------------------------------------------------------

λ [0] definitions: tail head add sub
λ [1] 25
λ [2] Goodbye.
